src/renderer/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -53,13 +47,12 @@ const Dashboard: React.FunctionComponent<IOwnProps & IInjectedProps &  Pick<IRoo
     const [notification, setNotification] = useState<INotificationState>(HiddenNotification);
 
     const onAddNewValidator = (): void => {
-        // TODO - implement
-
-        props.storeAddValidator(true);
-        props.history.push(Routes.ONBOARD_ROUTE_EVALUATE(OnBoardingRoutes.SIGNING));
-        // eslint-disable-next-line no-console
-        console.log("Add new validator");
-    };
+        
+        props.history.push({
+            pathname: Routes.ONBOARD_ROUTE_EVALUATE(OnBoardingRoutes.SIGNING),
+            state: {addValidator: "inProgress"}
```

Comment:
Can You rename this to like "isRegisterFlow": true or "isAddValidatorFlow": true?